testnet/stacks-node/Cargo.toml

```diff
@@ -8,18 +8,26 @@ edition = "2018"
 lazy_static = "1.4.0"
 pico-args = "0.3.1"
 rand = "=0.7.2"
-reqwest = { version = "0.10", features = ["blocking", "json", "native-tls-vendored"] }
 secp256k1 = { version = "0.11.5" }
 serde = "1"
 serde_derive = "1"
 serde_json = { version = "1.0", features = ["arbitrary_precision", "raw_value"] }
 stacks = { package = "blockstack-core", path = "../../." }
 toml = "0.5.6"
+async-h1 = "1.1"
```

Comment:
Please downgrade to `=1.0`.  `1.1` does not build for me.

Reply:
Also, why does the Stacks node need `async-h1`?

Reply:
Ah, I see we need it for Prometheus.  If that's the only thing we need it for, then please guard this (and `async-std`) with the appropriate feature flag.

Reply:
async-std is required for executing the futures handling the bitcoin rpc requests

Reply:
Gotcha.  But, please downgrade to `=1.0`.

Reply:
Addressed with